DXRMapFixups/DeusEx/Classes/DXRFixupVandenberg.uc

```diff
@@ -103,15 +103,17 @@ function PreFirstEntryMapFixes()
             hoverHint.SetBaseActor(jock);
         }
 
-        foreach AllActors(class'#var(prefix)SecurityBot2',sb,'enemy_bot') {
-            hoverHint = class'DXRHoverHint'.static.Create(self, "MJ12 Security Bot " $ ++botNum, sb.Location, sb.CollisionRadius*1.11, sb.CollisionHeight*1.11, sb.Name);
-            hoverHint.SetBaseActor(sb);
-            hoverHint.VisibleDistance = 15000;
-        }
-        botNum = 0;
-        foreach AllActors(class'#var(prefix)MilitaryBot',mb,'enemy_bot') {
-            hoverHint = class'DXRHoverHint'.static.Create(self, "MJ12 Military Bot " $ ++botNum, mb.Location, mb.CollisionRadius*1.1, mb.CollisionHeight*1.1, mb.Name);
-            hoverHint.SetBaseActor(mb);
+        foreach AllActors(class'#var(prefix)Robot',bot,'enemy_bot') {
+            if (bot.IsA('SecurityBot2')) {
```

Comment:
you forgot the `#var(prefix)` here and on the one below, also generally I prefer using casts to check type instead of the IsA function, since if you have a typo in the classname then the IsA function won't give an error at all, but casting will give an error at compile time
```suggestion
            if (#var(prefix)SecurityBot2(bot) != None) {
```